src/modules/menu/components/help/MenuHelpSection.vue

```diff
@@ -0,0 +1,157 @@
+<script setup>
+import { computed, ref } from 'vue'
+import { useI18n } from 'vue-i18n'
+import { useStore } from 'vuex'
+
+import { languages } from '@/modules/i18n/index.js'
+import FeedbackButton from '@/modules/menu/components/help/feedback/FeedbackButton.vue'
+import HelpLink from '@/modules/menu/components/help/HelpLink.vue'
+import MoreInfo from '@/modules/menu/components/help/MoreInfo.vue'
+import ReportProblemButton from '@/modules/menu/components/help/ReportProblemButton.vue'
+import MenuSection from '@/modules/menu/components/menu/MenuSection.vue'
+import AppVersion from '@/utils/components/AppVersion.vue'
+import ModalWithBackdrop from '@/utils/components/ModalWithBackdrop.vue'
+
+const dispatcher = { dispatcher: 'MenuHelpSection.vue' }
+
+const emits = defineEmits(['openMenuSection'])
+
+const sectionId = ref('helpSection')
```

Comment:
can be a simple value, I wasn't sure about giving a simple value in the `defineExpose`, but the doc states that it will unwrap any `ref` anyway, so need for a `ref` there